Backend/asrt/asrt/settings.py

```diff
@@ -89,7 +89,7 @@
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'project_taco_db',
         'USER': 'postgres',
-        'PASSWORD': 'lambtex',
+        'PASSWORD': 'paschal2020',
```

Comment:
Hi @paschalsabiti and @lambert3 ,

I hope these passwords that you put in here are not used anywhere else.
Ideally these details should be picked up from an environment. This will do for now though...

Reply:
Hi @RonKbS ,
We put all our environments in the gitignore file but we still view the passwords on the remote repo. How can we prevent this from happening.